### PR TITLE
Added styling using css module following Zelpin

### PR DESCRIPTION
**Pull request by sajibarafatsiddiqui**
- Added all necessary JSX elements to the project .
- Followed Zeplin's design (this includes all elements + the same styling)
- Used the CSS rules detailed in Zeplin to style the website.
Thanks

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,23 +1,50 @@
...
+            <button type="button" onClick={() => {}}>Comments</button>
+            <button
+              type="submit"
+              onClick={() => {
+                dispatch(removeBook(bookProp.item_id));
+              }}
+            >
+              remove
+            </button>
+            <button type="button" onClick={() => {}}>Edit</button>
```
Comment (Gambit142) [inline src/components/Book.js]: - [ ] Kindly make your outlook for each book look identical to that of Zeplin's template. In the requirements, all features in the template that have not been implemented till now should be used as a placeholder. For the individual books, kindly include space for the category of the book. The category of the book appears above the title of the book. Kindly implement this change. 👍 😎 

### Your Design

image

### Zeplin Design

image

File: src/components/InputBook.jsx [hunk 2]
```diff
@@ -32,17 +35,17 @@ const InputBook = () => {
...
           placeholder="Author"
           onChange={(e) => {
             setAuthor(e.target.value);
           }}
         />
-        <button type="submit">Submit</button>
+        <button type="submit" className="blue-button bold">Submit</button>
       </form>
-    </>
+    </div>
```
Comment (Gambit142) [inline src/components/InputBook.jsx]: - [ ] Kindly make the design of the form section identical to that of the Zeplin template. Please pay attention to the width of the form inputs and the width of the button. Also, pay attention to the spacing between the book container and the form and also the spacing between elements in the form. Kindly make your design look identical to that of Zeplin's template.😎 💻 
 
#### Hint: I would suggest you use a `grid` layout to make it responsive.

### Your Design

image


### Zeplin Template

image

File: src/styles/Book.module.css [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  margin-block-end: var(--margin-divisor);
+  border: 1px solid #e8e8e8;
+}
+
+.book,
+.book__details {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```
Comment (Gambit142) [inline src/styles/Book.module.css]: - [ ] I notice you repeated the rule `display: flex;` more than twice in your stylesheet. One of the best practices in CSS is to ensure that you don't make repeat style declarations in style sheets. I would advise creating a class that contains the declaration and using the class on JSX you desire to implement the style on. For example:

```css
.d-flex {
  display: flex;
}
```

Kindly implement this change on all declarations that are repeated more than twice in your style sheets. 👍 😎 You can read this [article](https://speckyboy.com/good-bad-css-practices/)

#### You could also use the CSS variable way to make your code clean. Read more on that in this [article](https://www.w3schools.com/css/css3_variables.asp)

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @sajibarafatsiddiqui 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Good use of CSS modules in react components ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/styles/Book.module.css (hunk 3 above)
Comment (sajibarafatsiddiqui) [reply src/styles/Book.module.css]: Changed it accordingly

File: src/components/InputBook.jsx (hunk 2 above)
Comment (sajibarafatsiddiqui) [reply src/components/InputBook.jsx]: Changed it accordingly

File: src/components/Book.js (hunk 1 above)
Comment (sajibarafatsiddiqui) [reply src/components/Book.js]: Changed it accordingly

File: src/components/InputBook.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            value={author}
+            placeholder="Author"
+            onChange={(e) => {
+              setAuthor(e.target.value);
+            }}
+            size="40"
+          />
+          <button type="submit" className={`blue-button bold ${styles.btn_width}`}>ADD BOOK</button>
+        </form>
+      </div>
```
Comment (Owusu-Desmond) [inline src/components/InputBook.js]: - [ ] While working on the changes the previous code reviewer requested about adjusting the spacing between the book container and the form and also the spacing between elements in the form, Your project is having horizontal overflow and your `Add Book` button is not showing on the page unless the page is scrolled horizontally. Please make sure you remove the horizontal overflow and make the design looks exactly like the Zeplin template:relaxed:

image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @sajibarafatsiddiqui , :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/InputBook.js (hunk 4 above)
Comment (sajibarafatsiddiqui) [reply src/components/InputBook.js]: Hi Desmond, thanks for pointing it out. I made that form responsive now.

**Review by frank1738 — APPROVED** [review]
Hi @sajibarafatsiddiqui , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter errors
✅ App is designed as expected 👍 


## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**